src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.shared.cs

```diff
@@ -19,6 +19,8 @@ namespace CommunityToolkit.Maui.Core.Views;
 /// </summary>
 public partial class MediaManager
 {
+	internal static MediaOptions DefaultMediaOptions { get; set; } = new();
```

Comment:
Here's is fine to be mutable, since users may want to change how options based on the `MediaSource`. But, it does make sense on the current implementation? The `MediaOptions` is only configurable on `AppBuilder`, so it shouldn't change after app startup, right?

Reply:
Agreed. Let's make these `{ get; private set;}` for now to follow the same architecture in `CommunityToolkit.Maui.Options`: https://github.com/CommunityToolkit/Maui/blob/d6d6e187b4f6136558bc3be2d2ecd8cca06fc96f/src/CommunityToolkit.Maui/Options.cs#L19-L22

We can always make it a `public set;` in the future without breaking changes.

Reply:
Do we want to add bindable properties so we can use this at runtime for changes with `MediaSource` ?